projects/canopy/src/lib/promo-card/promo-card.notes.ts

```diff
@@ -1,6 +1,4 @@
 export const notes = `
-# Promo Card List Component
```

Comment:
this gives a weird duplication of headers so i've allowed the default 'Promo card' to filter through

Reply:
👍 that's what we've done in others too